happy/State.py

```diff
@@ -181,6 +181,17 @@ def getNodeInterfaceAddresses(self, interface_id, node_id=None, state=None):
             return []
         return node_interface["ip"].keys()
 
+    def getNodeAddrMatchPrefix(self, node_id, interface, prefix):
```

Comment:
This needs API documentation.

Reply:
Suggest renaming this `getNodeAddrMatchingPrefix`.

Reply:
done